main/pagespeed-site.conf.template

```diff
@@ -17,5 +17,4 @@ pagespeed MapOriginDomain $PROXY_ADDR https://cdn2-$DOMAIN $DOMAIN;
 pagespeed Disallow "*/cruicons.woff?10301097";
 
 pagespeed AdminPath /pagespeed_admin;
-location ~ ^/pagespeed_admin { allow 10.16.0.0/16; deny all; }
-
+location ~ ^/pagespeed_admin { allow 127.0.0.1; allow 10.16.18.120; allow 10.16.0.0/16; deny all; }
```

Comment:
Isn't `allow 10.16.18.120` repetitive with `allow 10.16.0.0/16`?

Reply:
Yes, This was a test I was trying to figure out why it was not working. That's when I discover that the SG was blocking the IP range because it only allowed a set of ports and a set of other SGs. I'll remove the dup and fix the issue by adding an entry on the SG (sg-3ddef259) to allow 10.16.0.0/16 on port 80 and 443

![screen shot 2018-03-07 at 9 18 24 am](https://user-images.githubusercontent.com/6875635/37097100-9bd9c238-21e8-11e8-9fb5-d0750f67aa30.png)